manifests/kots-config.yaml

```diff
@@ -442,6 +442,15 @@ spec:
             If you don't provide an Allocation ID, a random domain will be assigned to your load balancer. You can find some information about how to use the domain you configured for CARTO platform in the [AWS ELB documentation](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/using-domain-names-with-elb.html).
           recommended: true
 
+        # Ingress testing mode
+        - name: enableIngressTestingMode
+          title: Ingress testing mode
+          when: '{{repl (ConfigOptionEquals "accessToCartoModeK8s" "accessToCartoModeK8sCustom") }}'
```

Comment:
I thought we were going to leave it available for all modes.

I don't have a strong opinion but you should also think about the single instance that places a load balancer in front of the instance :see_no_evil:

Reply:
It's configured right now to be shown in both embedded cluster and existing cluster 🤔 Maybe the `accessToCartoModeK8s` is confusing as it handles both the embedded cluster and the existing cluster

Reply:
😓 You were right, I missed one part of the conditional 😅 Fixed! Thanks for checking it!